spk/adminer/src/conf/resource

```diff
@@ -0,0 +1,48 @@
+{
+  "webservice": {
+    "services": [
+      {
+        "service": "adminer",
+        "display_name": "Adminer",
+        "type": "apache_php",
+        "root": "adminer",
+        "backend": 2,
+        "icon": "app/images/adminer-{0}.png",
+        "php": {
+          "profile_name": "adminer Profile",
+          "profile_desc": "PHP Profile for adminer",
+          "backend": 8,
+          "open_basedir": "/var/services/web_packages/adminer:/tmp:/var/services/tmp",
```

Comment:
I highly recommend removing `open_basedir`. I might found a bug in the DSM7 where the first package that sets the open_basedir will also wrongly apply for any package installed after it. I'm in contact with Synology but their communication is slow... Since the DSM7 is further locked down I don't see a security issue not having it. but would be nice if it worked correctly.

Reply:
<pre>
AH01071: Got error 'PHP message: PHP Warning: Unknown: open_basedir** restriction in effect. 
File(/volume1/web_packages/<b>wallabag</b>/web/app.php) is not within the allowed path(s): 
(/var/services/web_packages/<b>adminer</b>:/tmp:/var/services/tmp) in Unknown on line 0
PHP message: PHP Warning: Unknown: failed to open stream: Operation not permitted in Unknown on line 0
Unable to open primary script: /var/services/web_packages/wallabag/web/app.php (Operation not permitted)'
</pre>

Reply:
@publicarray thanks 👍